### DISCUSSION TITLE
External web-ui access?

### DISCUSSION THREAD

**Discussion by isensez** [bionic-gpt/bionic-gpt#319] [Announcements]
### Describe your issue

Running locally, the webui works perfectly fine, everything works well on a local setup.

When running bionic-gpt on a dedicated server, and trying to access it via the devices local ip, ie:192.168.1.2, without a port and with port, it redirects to a localhost address instead.

While the webui itself isnt directly accessible, the envoy admin page for example, ends up initially loading, although just that, pressing the link to try to get into it also fails when trying to externally access it.

### Describe your setup/startup scripts

setup bionic-gpt via docker-compose for dev or kubernetes for prod

### Steps to reproduce

Host bionic-gpt on a server.
Try to access the webui from an external device.

### What was the expected result?

Expected for webui to be accessible from the external IP of the device (ie 192.168.1.2) along with localhost.

### Any additional information

This happens in all setups, be it docker-compose for dev or kubernetes for prod.

**Comment by robertsmaoui** [comment]
@isensez 
in docker compose yml file replace localhost by your ip

**Reply by isensez** [reply]
Thanks! Really appreciated
I got the web UI reachable from an external source, but promps do not work with the default model atm.
"Error occurred while generating."
I will be playing around with some more settings.

Using the docker-compose ollama version

**Reply by robertsmaoui** [reply]
Have you installed ollama ? (in linux curl https://ollama.ai/install.sh | sh  )
after installation, ollama run llama2 
this file mentioned in documentation(curl -O https://raw.githubusercontent.com/bionic-gpt/bionic-gpt/main/docker-compose-ollama.yml) use only  llama2 regardless of the name of the model  (in model setup in bionic)

**Reply by isensez** [reply]
Yes ollama was installed but something may have been messed up in the process.
I rebuild the installation and got the enterprise version running with external web access instead.
Appreciate the prompt reply and help in this process!
